http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala

```diff
@@ -1005,74 +1005,26 @@ class LeoPubsubMessageSubscriber[F[_]](
       // _initial_ GKE call synchronous to the pubsub processing so we can nack the message on
```

Comment:
does this comment need to be updated with the new autopilot logic?

Reply:
I think it still applies since with autopilot, we're still creating the cluster when there's no existing clusters